client/js/components/GameListItem.vue

```diff
@@ -102,29 +102,50 @@ export default {
 </script>
 <style scoped lang="scss">
 .list-item {
-  margin-left: 0;
-  margin-right: 0;
-  text-align: left;
+  margin: 0;
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+  flex-direction: column;
   align-items: center;
+  padding-top: .5rem;
   & .game-name {
     font-weight: 600;
+    font-size: 1.10em;
+    text-align: left;
+  }
+  & p {
+      line-height: 1;
+      margin: 3px auto;
+      
   }
 }
+
+.list-item--inner-text {
+  display: flex;
+  padding-bottom: 1rem;
+  padding-top: .25rem;
+}
+
 .list-item-button {
   display: flex;
   align-items: center;
   justify-content: center;
+  margin-top: 0;
+  padding-top: .5rem;
 }
```

Comment:
I know you didn't add this class, but this can also probably go in to `.list-item` as:
```
&__button {
  display: flex;
  align-items: center;
  justify-content: center;
  margin-top: 0;
  padding-top: .5rem;
}
```